library/tinkerpop/src/test/java/uk/gov/gchq/gaffer/tinkerpop/GafferPopEdgeTest.java

```diff
@@ -28,18 +30,24 @@
 import java.util.Iterator;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertSame;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
 
 public class GafferPopEdgeTest {
+
+    public static String source = "source";
```

Comment:
Sorry, I should have mentioned, these are usually formatted like so:
```java
public static final String SOURCE = "source";
```
See GafferPopGraphTest:
https://github.com/gchq/Gaffer/blob/ecc9f4487b1106f3ceffde3031e2018f776f96b7/library/tinkerpop/src/test/java/uk/gov/gchq/gaffer/tinkerpop/GafferPopGraphTest.java#L47-L55